frontend/src/translations/result/en.json

```diff
@@ -542,5 +542,15 @@
   "first.task.community.description": "We will make a campaign to let our community know that you have an issue to be solved",
   "first.task.community.title": "Send to our community",
   "first.task.next": "Next",
-  "first.task.back": "Back"
+  "first.task.back": "Back",
```

Comment:
@longj724 Same here. You don't need to change the result files, they're managed by our translation tool, so please don't add the result files on this PR.